sanity/doctors/ParcelDoctor.js

```diff
@@ -94,10 +94,25 @@ export class ParcelDiagnosis extends Diagnosis {
   constructor(faultyParcels) {
     super()
     this.faultyParcels = faultyParcels
+    this.estateDiagnosis = new EstateDiagnosis()
   }
 
-  getFaultyAssets() {
-    return this.faultyParcels
+  async getFaultyAssets() {
+    const faultyAssets = [...this.faultyParcels]
```

Comment:
why are we doing this? is `faultyParcels` a `Set` ?

Reply:
No, but I need to create a new object and not change `this.FaultyParcels`